### PR TITLE
add code to where you can go to the details page of an opportunity th…

### PR DESCRIPTION
**Pull request by droopy25**
…at has been applied for to be able to review qualifications

### CODE REVIEW

**Comment by Chef-Code** [comment]
OK I'm going to review this pull request

File: Coop_Listing_Site/Coop_Listing_Site/Web.config [hunk 1]
```diff
@@ -39,9 +39,16 @@
...
+    <security>
+      <requestFiltering>
+        <requestLimits maxAllowedContentLength="1073741824" />
+      </requestFiltering>
+    </security>
   </system.webServer>
-  -->
+  <system.web>
+    <httpRuntime maxRequestLength="1048576" />
+  </system.web>
```
Comment (Chef-Code) [inline Coop_Listing_Site/Coop_Listing_Site/Web.config]: @droopy25 Why did we have this commented out in the first place again?

File: Coop_Listing_Site/Coop_Listing_Site/Web.config [hunk 2]
```diff
@@ -15,10 +15,10 @@
   </connectionStrings>
 
   <!-- Force SSL/TLS the first time the site is visited, and for all subsequent visits for the next year using HTTP Strict Transport Security (HSTS) -->
-  <!-- Uncomment when deploying to the web server
+  <!-- Uncomment when deploying to the web server-->
```
Comment (Chef-Code) [inline Coop_Listing_Site/Coop_Listing_Site/Web.config:18]: @droopy25 Why did we have this commented out in the first place again? I know the 

> when deploying to the web server...part

but was there a reason we were waiting?

**Comment by Chef-Code** [comment]
@Kethsar  or @BenMiddleton or anyone that knows the answer, you should approve this pull request.  I'm not confident enough in my decision.  It looks fine, but again.

**Comment by Kethsar** [comment]
@Chef-Code if you read the part above it, the commented out XML forces the connection to use SSL. Unless the server, including the local one, has SSL enabled, it would effectively make it impossible for someone to get to the server.

IIS Express has SSL disabled by default. If that were uncommented, you guys would have trouble accessing your local server until you enabled SSL for IIS Express. While not particularly hard, it would cause the extra step of having to go to the SSL port instead of the default one, so always keep it commented out unless deploying to a server. Never run the site locally with it uncommented either.

**Comment by Chef-Code** [comment]
@Kethsar So I take it that we are not accepting this pull request then???

**Pull request merged**